### DISCUSSION TITLE
如何获取下划线内容的选区？

### DISCUSSION THREAD

**Discussion by Blank-Tea** [wangeditor-next/wangEditor-next#412] [General]
## 问题描述

请问使用wangEditor-next/vue如何获取富文本中所有下划线内容的选区；尝试使用editor.nodes好像没有能查u标签的;请问有没有其他方法能实现？

**Comment by cycleccc** [comment]
没有对应的 API，选区可以自己推出来，根据 editor.children 可以推算出有下划线属性的元素在哪个层级。

**Comment by cycleccc** [comment]
当前没有直接 API，可根据 editor.children 推导带下划线节点的选区。本讨论现关闭。
